gui/frontend/src/search/search.html

```diff
@@ -238,7 +238,8 @@
                 </div>
                 <table ng-if="field !== 'images'" class="table table-hover table-condensed"
                     ng-init="fieldType = search.getFieldType(field)">
-                    <tr ng-repeat="bucket in agg.buckets track by $index" ng-click="search.doNavigate(field, bucket.key)">
+                    <!--<tr ng-repeat="bucket in agg.buckets track by $index" ng-click="search.doNavigate($event, field, bucket.key)">-->
+                    <tr ng-repeat="bucket in agg.buckets track by $index" ng-click="search.doNavigate($event, field, bucket.key)">
```

Comment:
Yes, $event is an angular builtin. I updated calls to `doNavigate` in app-specific template as well.